src/reflect/scala/reflect/internal/Types.scala

```diff
@@ -9,7 +9,7 @@ package internal
 
 import scala.collection.{ mutable, immutable }
 import scala.ref.WeakReference
-import mutable.ListBuffer
+import mutable.{ BitSet, ListBuffer }
```

Comment:
If this isn't used, we should import it.

Reply:
I was trying to use a `BitSet` in the `suspendingTypeVars` method, but I have found some off behaviour, maybe a bug, that I will show elsewhere.